### ISSUE TITLE
Emojis can break the rendering

### ISSUE THREAD

**Issue by Glaived** [ayonious/console-table-printer#389]
console-table-printer v2.9.0

```js
const bundle = new Table({
  title: 'Bundle (BUNDLENAME)',
  columns: [
    { name: 'Weapon' },
    { name: 'Chroma' },
    { name: 'Quality' },
    { name: 'Price' },
  ],
})

bundle.addRows([
  {
    Weapon: '👍🏻',
    Chroma: '✅',
    Quality: 'Deluxe',
    Price: '1 775 VP',
  },
])

bundle.printTable()
```

Result:
![image](https://user-images.githubusercontent.com/1835170/127753629-9a6166cd-15fa-40c7-98f3-9d0f359df7b7.png)

**Comment by ayonious** [comment]
Very nice catch.

Unfortunately this is a basic limitation. I calculate the character width based on wcwidth.c 
https://www.cl.cam.ac.uk/~mgk25/ucs/wcwidth.c

As emojis appeared lot later with a little complicated rules I need to fix this basic algorithm which could be a little difficult. For example some skin tone signs are supposed to be 0 width and at same time could be 1 width if used individually the whole logic is a little complicated. And this whole logic should be fixed in this underlying dependency.
https://www.npmjs.com/package/simple-wcswidth

PRs are welcome for any simple wcs width calculation for emojis. This might unfortunately take some time for me to fix in this underlying package. But Im thinking of some custom instruction that can be passed during Table instance creation which the user defines to make this Problem go away in a forced way.

For example new Table({ wcswidth: { '👍🏻': 1, '✅': 2 } }); 
to make sure the width of those chars to be user defined.

Let me know what you guys think.

**Comment by ArjixWasTaken** [comment]
I think having it be custom defined is the best method.
After all, it not only fixes the issue at hand, it also gives us more customizability.

**Comment by ayonious** [comment]
This has been fixed in 2.11.1
